extensions/trackedfile/src/main/java/io/deephaven/extensions/trackedfile/TrackedSeekableChannelsProviderPlugin.java

```diff
@@ -17,20 +16,24 @@
  * {@link SeekableChannelsProviderPlugin} implementation used for reading files from local disk.
  */
 @AutoService(SeekableChannelsProviderPlugin.class)
-public final class TrackedSeekableChannelsProviderPluginImpl implements SeekableChannelsProviderPlugin {
+public final class TrackedSeekableChannelsProviderPlugin implements SeekableChannelsProviderPlugin {
+
+    static final String FILE_URI_SCHEME = "file";
+
     @Override
     public boolean isCompatible(@NotNull final URI uri, @Nullable final Object object) {
-        return object == null && (uri.getScheme() == null || uri.getScheme().equals(ParquetFileReader.FILE_URI_SCHEME));
+        return FILE_URI_SCHEME.equals(uri.getScheme());
     }
 
     @Override
-    public SeekableChannelsProvider impl(@NotNull final URI uri, @Nullable final Object object) {
+    public SeekableChannelsProvider createProvider(@NotNull final URI uri, @Nullable final Object object) {
         if (!isCompatible(uri, object)) {
             if (object != null) {
                 throw new IllegalArgumentException("Arguments not compatible, provided non null object");
             }
             throw new IllegalArgumentException("Arguments not compatible, provided uri " + uri);
         }
+        // TODO I am confused which constructor should I use, this or getInstance()??
```

Comment:
Good Q - I don't think there's much value in `TrackedSeekableChannelsProvider.getInstance()`, I would just get rid of that one.